src/index.html

```diff
@@ -4,9 +4,281 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" >
+        <img
+          src="/images/logo.png"
+          class="header__logo"
+          alt="logo"
+        >
+      </a>
+      <nav class="navbar">
+        <ul class="navbar__list">
+          <li class="navbar__item">
+            <a href="#" class="navbar__link navbar__link--current is-active">Apple
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link">Samsung
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link">Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a
+              href="#"
+              class="navbar__link"
+              data-qa="nav-hover">Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="navbar__item">
+            <a href="#" class="navbar__link" >Гаджеты
+            </a>
```

Comment:
мені здається так код більш читабельний
```suggestion
            <a href="#" class="navbar__link" >
              Гаджеты
            </a>
```

Reply:
Так, ти правий

Reply:
Тут також